maps/torch/items/clothing/solgov-suit.dm

```diff
@@ -474,9 +474,11 @@
 	icon_state = "helm_explorer"
 	item_state = "helm_explorer"
 	sprite_sheets = list(
-		SPECIES_SKRELL = 'maps/torch/icons/mob/skrell/onmob_head_solgov_skrell.dmi'
```

Comment:
Why was skrell helmet removed?

Reply:
I brought it back the wrong way.
You haven't seen shit.